src/Elements/Comment.react.js

```diff
@@ -64,11 +178,24 @@ class Comment extends Component {
 
 Comment.propTypes = {
     avatarSrc: PropTypes.string,
+    canEdit: PropTypes.bool,
+    canRemove: PropTypes.bool,
     className: PropTypes.string,
     detailsPosition: PropTypes.oneOf([ 'left', 'right' ]),
+    isEditable: PropTypes.bool,
     name: PropTypes.string,
+    onRemove: PropTypes.func,
+    onSaveEdit: PropTypes.func,
     style: PropTypes.object,
+    text: PropTypes.string,
```

Comment:
@morethanfire For edit mode, I had to expose a prop where the contents on the comment come in as a string (whereas the previous read-only version used `children` prop to accept a JSX node, which is better and more flexible.  Is there a way to render JSX to a string and avoid having to do this?  Any thoughts?